azure-pipelines.yml

```diff
@@ -233,8 +233,8 @@ jobs:
 
     - template: etc/ci/azure-posix.yml
       parameters:
-          job_name: macos12_cpython_latest_from_pip
-          image_name: macos-12
+          job_name: macos14_cpython_latest_from_pip
+          image_name: macos-14
```

Comment:
Could you bump only to macos-13 for now https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md ?

Reply:
We already have them, see https://github.com/aboutcode-org/scancode-toolkit/pull/4089/files/20663f45f0cc0743e179117c28914387598e177b#diff-7915b9b726a397ae7ba6af7b9703633d21c031ebf21682f3ee7e6a4ec52837a5L245
Basically we had the following runners: macos-12, macos-13, now we have macos-13, macos-14